.gitignore

```diff
@@ -1,3 +1,5 @@
+config/initializers/secret_token.rb
```

Comment:
hmm I guess rails 4.2 is now using config/secrets.yml I'll do that locally to and fix that